src/main/resources/assessment-config/finance-and-id/finance-and-id-assessment-v1.yml

```diff
@@ -0,0 +1,169 @@
+version: 1
+pathway: "FINANCE_AND_ID"
+genericAssessmentVersion: 1
+pages:
+  - id: HAS_BANK_ACCOUNT
+    questions:
+      - id: HAS_BANK_ACCOUNT
+        title: "Does the person in prison have a bank account?"
+        type: "RADIO"
+        options:
+          - id: "YES"
+            displayText: "Yes"
+          - id: "NO"
+            displayText: "No"
+          - id: "NO_ANSWER"
+            displayText: "No answer provided"
+    nextPageLogic:
+      - questionId: HAS_BANK_ACCOUNT
+        answers:
+          - "@class": "StringAnswer"
+            answer: "YES"
+        nextPageId: WHAT_ID_DOCUMENTS
+      - questionId: HAS_BANK_ACCOUNT
```

Comment:
Might be worth explicitly listing all the possible other answers (i.e. NO and NO_ANSWER) so it validates that the answer given is one we're expecting